recommender/engines/autoencoders/training/model_validation_step.py

```diff
@@ -23,9 +21,10 @@ def check_performance(metrics, max_loss_score: int) -> None:
         metrics: data from the AEModelEvaluationStep,
         max_loss_score: threshold used for validation.
     """
-
     for collection_name, metric in metrics.items():
-        for split, loss in metric.items():
+        for split, data in metric.items():
+            loss = data[LOSS]
+            # acc = data[ACCURACY]
```

Comment:
if we dont use accuracy in the model validation step we should just not include it, bascially, just delete the commented line

Reply:
Also, it might be a good idea to include it during model validation, but's im not sure its worht it now since the new AE changes are coming